src/main/java/org/sonarlint/intellij/tasks/InformationFetchTask.java

```diff
@@ -33,15 +33,17 @@
 import org.sonarsource.sonarlint.core.client.api.connected.ServerConfiguration;
 import org.sonarsource.sonarlint.core.client.api.connected.WsHelper;
 import org.sonarsource.sonarlint.core.client.api.exceptions.UnsupportedServerException;
+import org.sonarsource.sonarlint.core.notifications.SonarQubeNotifications;
 
-public class OrganizationsFetchTask extends Task.Modal {
+public class InformationFetchTask extends Task.Modal {
   private static final Logger LOGGER = Logger.getInstance(ConnectionTestTask.class);
   private final SonarQubeServer server;
   private Exception exception;
-  private List<RemoteOrganization> result;
+  private List<RemoteOrganization> organizations;
+  private boolean notificationsSupported = false;
 
-  public OrganizationsFetchTask(SonarQubeServer server) {
-    super(null, "Fetch Organizations From SonarQube Server", true);
+  public InformationFetchTask(SonarQubeServer server) {
```

Comment:
For the record, I'm not a fan of "Information", but don't have a better idea now